src/Goerli-DssSpell.t.sol

```diff
@@ -25,6 +25,223 @@ interface DenyProxyLike {
+                pip == 0xdF8474337c9D3f66C0b71d31C7D3596E4F517457 ||
+                pip == 0x57A00620Ba1f5f81F20565ce72df4Ad695B389d7) {
+                continue;
+            }
+
+            assertEq(OsmAbstract(pip).wards(ORACLE_WALLET01), 0);
+            assertEq(OsmAbstract(pip).wards(ORACLE_WALLET02), 0);
+        }
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        for(uint256 i = 0; i < ilks.length; i++) {
+            uint256 class = reg.class(ilks[i]);
+            if (class != 1) { continue; }
+
+            address pip = reg.pip(ilks[i]);
+            // skip USDC, TUSD, PAXUSD, GUSD
+            if (pip == 0x838212865E2c2f4F7226fCc0A3EFc3EB139eC661 ||
+                pip == 0x0ce19eA2C568890e63083652f205554C927a0caa ||
+                pip == 0xdF8474337c9D3f66C0b71d31C7D3596E4F517457 ||
+                pip == 0x57A00620Ba1f5f81F20565ce72df4Ad695B389d7) {
+                continue;
+            }
+
+            assertEq(OsmAbstract(pip).wards(ORACLE_WALLET01), 1);
+            assertEq(OsmAbstract(pip).wards(ORACLE_WALLET02), 1);
+        }
+    }
+
+    function test_oracle_list() public {
+        address ORACLE_WALLET01 = 0x4D6fbF888c374D7964D56144dE0C0cFBd49750D3;
+        address ORACLE_WALLET02 = 0x1f42e41A34B71606FcC60b4e624243b365D99745;
+
+        // validate the spell does what Oracle's wanted
+        // https://discord.com/channels/893112320329396265/897479589171986434/960944481157390336
+        assertEq(OsmAbstract(0xF15993A5C5BE496b8e1c9657Fd2233b579Cd3Bc6).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0x2BA78cb27044edCb715b03685D4bf74261170a70).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0xc3d677a5451cAFED13f748d822418098593D3599).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0x94588e35fF4d2E99ffb8D5095F35d1E37d6dDf12).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0xF953cdebbbf63607EeBc556438d86F2e1d47C8aA).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0x6Fb18806ff87B45220C2DB0941709142f2395069).wards(ORACLE_WALLET01), 0);
+        // assertEq(OsmAbstract(0x57A00620Ba1f5f81F20565ce72df4Ad695B389d7).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0xCB772363E2DEc06942edbc5E697F4A9114B5989c).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0x75B4e743772D25a7998F4230cb016ddCF2c52629).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0x5AD3A560BB125d00db8E94915232BA8f6166967C).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0x001eDD66a5Cc9268159Cf24F3dC0AdcE456AAAAb).wards(ORACLE_WALLET01), 0);
+        assertEq(OsmAbstract(0xDe112F61b823e776B3439f2F39AfF41f57993045).wards(ORACLE_WALLET01), 0);
+        // assertEq(OsmAbstract(0xdF8474337c9D3f66C0b71d31C7D3596E4F517457).wards(ORACLE_WALLET01), 0);
+        // assertEq(OsmAbstract(0xdF8474337c9D3f66C0b71d31C7D3596E4F517457).wards(ORACLE_WALLET01), 0);
```

Comment:
why are these commented out?

Reply:
These represent the original ask from Oracles (see discord), but the original ask was wrong (as stables and RWA don't have `wards`).